src/adhocracy_mercator/adhocracy_mercator/scripts/manage_badges.py

```diff
@@ -0,0 +1,113 @@
+"""Assign badges to proposals.
+
+The functions of that script are registered as console script in
+setup.py.
+
+"""
+
+import inspect
+import argparse
+import transaction
+import json
+
+from pyramid.paster import bootstrap
+from pyramid.traversal import find_resource
+from adhocracy_core.resources.badge import IBadgeAssignment as BadgeResource
+from adhocracy_core.sheets.badge import IBadgeAssignment as BadgeSheet
+
```

Comment:
I was confused what BadgeSheet is exactly.
To seperate resource.badge.IBadgeAssignment and sheets.badge.IBadgeAssignment we import the full namesspace for the sheet: import adhocracy_core.sheets.badge.IBadgeAssignement